src/usr/lib/rsetup/cli/u-boot-menu.sh

```diff
@@ -132,3 +134,49 @@ rebuild_overlays() {
     mv "$old_overlays" "${old_overlays}_old"
     mv "$new_overlays" "$old_overlays"
 }
+
+enable_overlays() {
+    load_u-boot_setting
+    for i in "$@"
+    do
+        if [[ $i == *.disabled ]]
+        then
+            if [ -e "$U_BOOT_FDT_OVERLAYS_DIR/$i" ]
```

Comment:
```suggestion
            if [[ -e "$U_BOOT_FDT_OVERLAYS_DIR/$i" ]]
```
bash我们一律使用[[ ]]，这个更安全，也可以避免一些行为上的差异导致的问题